packages/functional-tests/tests/react-conversion/oauthResetPassword.spec.ts

```diff
@@ -21,6 +21,7 @@ test.describe('oauth reset password react', () => {
     resetPassword.react = resetPasswordReactFlag;
   });
 
+  /*Disabling test in stage until Fxa-8192 is worked on
```

Comment:
Should this skip if in the stage environment vs disable for all?

Reply:
Its failing in Stage and Prod both with the same error. Hence disabling worked better in my opinion. What do you say?

Reply:
I'd make the comment to mention prod too!